<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgDisReinvest</class>
 <widget class="QWidget" name="wdgDisReinvest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1102</width>
    <height>806</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/xulpymoney.qrc">
    <normaloff>:/xulpymoney/tools-wizard.png</normaloff>:/xulpymoney/tools-wizard.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_21">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="lblTitulo">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 128, 0);</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="grp">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QRadioButton" name="radRe">
                <property name="text">
                 <string>Reinvest simulation</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radDes">
                <property name="text">
                 <string>Disinvest simulation</string>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Operation data</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_12">
                   <item>
                    <widget class="QLabel" name="lblSimulacion"/>
                   </item>
                   <item>
                    <widget class="myQLineEdit" name="txtSimulacion">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>0</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_5">
                   <item>
                    <widget class="QLabel" name="lblValor"/>
                   </item>
                   <item>
                    <widget class="myQLineEdit" name="txtValorAccion">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>0</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <item>
                    <widget class="QLabel" name="label_3">
                     <property name="text">
                      <string>Bank comission</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="myQLineEdit" name="txtComision">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>0</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_11">
                   <item>
                    <widget class="QLabel" name="label_5">
                     <property name="text">
                      <string>Shares number</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="myQLineEdit" name="txtAcciones">
                     <property name="enabled">
                      <bool>false</bool>
                     </property>
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_3">
                   <item>
                    <widget class="QLabel" name="label_2">
                     <property name="text">
                      <string>Amount</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="myQLineEdit" name="txtImporte">
                     <property name="enabled">
                      <bool>false</bool>
                     </property>
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="cmdOrder">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Add order annotations</string>
                </property>
                <property name="icon">
                 <iconset resource="../images/xulpymoney.qrc">
                  <normaloff>:/xulpymoney/add.png</normaloff>:/xulpymoney/add.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="cmd">
                <property name="text">
                 <string>Make simulation</string>
                </property>
                <property name="icon">
                 <iconset resource="../images/xulpymoney.qrc">
                  <normaloff>:/xulpymoney/tools-wizard.png</normaloff>:/xulpymoney/tools-wizard.png</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="cmdGraph">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Show operations in a graph</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../images/xulpymoney.qrc">
           <normaloff>:/xulpymoney/report.png</normaloff>:/xulpymoney/report.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Select a price to evaluate the simulation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbPrices">
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabOps">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab_13">
        <attribute name="title">
         <string>Investment operations</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_24">
         <item>
          <widget class="mqtw" name="mqtwOps" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="Situac_3">
        <attribute name="title">
         <string>Investment current state</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_26">
         <item>
          <widget class="mqtw" name="mqtwCurrentOps" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Investment historical operations</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="mqtwObjects" name="mqtwHistoricalOps" native="true"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">myqlineedit</header>
  </customwidget>
  <customwidget>
   <class>mqtw</class>
   <extends>QWidget</extends>
   <header location="global">myqtablewidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>mqtwObjects</class>
   <extends>QWidget</extends>
   <header location="global">myqtablewidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/xulpymoney.qrc"/>
 </resources>
 <connections/>
</ui>
